repo2docker/contentproviders/zenodo.py

```diff
@@ -105,9 +146,13 @@ def _fetch(file_ref, unzip=False):
                         copytree(path.join(output_dir, d), output_dir)
                         shutil.rmtree(path.join(output_dir, d))
 
-        is_software = record["metadata"]["upload_type"] == "software"
-        only_one_file = len(record["files"]) == 1
-        for file_ref in record["files"]:
+        is_software = self._getfromdict(record, host["type"]).lower() == "software"
+        files = self._getfromdict(record, host["filepath"])
+
+        #
```

Comment:
Was there a comment here that got lost? If yes we should add it back, otherwise remove one of the blank lines and this line.